source/user-manual/reference/ossec-conf/wodle-vuln-detector.rst

```diff
@@ -57,7 +57,7 @@ interval
 Time between vulnerabilities detections.
 
 +--------------------+------------------------------------------------------------------------------------------------------------------------------------------+
-| **Default value**  | 1d                                                                                                                                       |
+| **Default value**  | 1m                                                                                                                                       |
 +--------------------+------------------------------------------------------------------------------------------------------------------------------------------+
 | **Allowed values** | A positive number that should contain a suffix character indicating a time unit, such as, s (seconds), m (minutes), h (hours), d (days). |
```

Comment:
Maybe `such as,` would be better as `such `? Maybe `, d (days)` would be better as `or d (days)`?